docs/PacketFence_Aerohive_Quick_Install_Guide.asciidoc

```diff
@@ -18,433 +18,264 @@ include::includes/global-attributes.asciidoc[]
 
-Click Ok to save your changes.
+Create or modify an SSID to add the Captive Portal fonction::
+* From the HiveManager, go to *CONFIGURATION -> SSIDS -> Select your SSID.*
+* From _SSID Access Security_ , Check *Enable Captive Web Portal*
+* Click *Save*
 
-Configure the Open SSID
-+++++++++++++++++++++++
-On the left menu, click on 'SSIDs'. Click the 'New' button, and configure the SSID like in the following screen capture:
+* From the HiveManager, go to *CONFIGURATION -> NETWORK POLICIES*, Select your policy
+* From _Configure Interfaces and User Access_, you will now have a _CWP_ symbol under _Authentication_.
+* Under _User Profile_ , click on *Add/Remove* and select the _default-profile_ (1) option to have the native VLAN as default
+* Click on *Save*
 
-image::docs/images/aerohive-ssid-public.png[scaledwidth="100%",alt="SSID open configuration"]
+image::docs/images/pfaerohive12.png[scaledwidth="100%",alt="Captive Portal SSID configuration",align=right]
 
-NOTE: Make sure to select 'Enable Captive Web Portal' when using Webauth.
 
-Click OK to save the SSID.
+'''
 
-Configure the Secure SSID
-+++++++++++++++++++++++++
-On the left menu, click on 'SSIDs'. Click the 'New' button, and configure the SSID like in the following screen capture:
+Push your configuration to your AP::
+* From _Configure and Update Devices_, check your AP in _Device to Update_
+* Click on _Update_
+* Select _update Devices_
+* From _HiveOS Number of devices running earlier versions of HiveOS_, select *Upgrade these devices to the latest version of HiveOS*
+* Click on _Update_
+* Wait until the date and time appears under _Update Status_
 
-image::docs/images/aerohive-ssid-secure.png[scaledwidth="100%",alt="SSID secure configuration"]
 
-NOTE: Make sure to select 'Enable Captive Web Portal' when using Webauth.
 
-Click OK to save the SSID.
+Step 2: Configure PacketFence
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Configure User Profile
-^^^^^^^^^^^^^^^^^^^^^^
-You will now need to create user profile matching your VLAN assignments, the "Default VLAN" have been created before to match the desired VLAN number.
+Configure the _Production_ interface to send the _Portal_.
+* Go to *CONFIGURATION -> NETWORK CONFIGURATION -> Interfaces*
+* Under _Logical Name_, click on your interface name,
+* _Additionnal listening daemon(s)_, Add *portal*
+* Click on *SAVE*
 
-Create user profile as follow:
 
-image::docs/images/aerohive-user-profile-reg.png[scaledwidth="100%",alt="User profile registration"]
+image::docs/images/pfaerohive13.png[scaledwidth="100%",alt="Captive Portal SSID configuration",align=center]
 
-image::docs/images/aerohive-user-profile-prod.png[scaledwidth="100%",alt="User profile production"]
 
-Configure SNMP
-^^^^^^^^^^^^^^
-You now need to configure the SNMP to add a valid read community string. Click on the 'Configure' pane, and on 'Advanced Configuration', then 'Management Policies', then 'SNMP Assignments'. Now click on 'New' and follow this configuration.
+'''
```

Comment:
It miss the Packetfence configuration side where you enable External Captive Portal in the switch configuration.